ansibullbot/utils/component_tools.py

```diff
@@ -17,6 +17,17 @@
 from ansibullbot.utils.systemtools import run_command
 
 
+
+def make_prefixes(filename):
+    # make a byte by byte list of prefixes for this fp
+    indexes = range(0, len(filename) + 1)
+    indexes = [1-x for x in indexes]
```

Comment:
@jctanner I'd be better communicating the intent if you were using range in reverse order of iteration:
```python
>>> fname = 'some_file'
>>> import itertools
>>> [fname[:x] for x in itertools.chain((None, ), range(-1, -len(fname), -1))]
['some_file', 'some_fil', 'some_fi', 'some_f', 'some_', 'some', 'som', 'so', 's']
```
I was puzzled about this `1-x` thing when I first saw it